provider/source/src/calendar/hijri.rs

```diff
@@ -32,7 +32,74 @@ impl crate::IterableDataProviderCached<CalendarHijriObservationalMeccaV1> for So
 impl DataProvider<CalendarHijriUmmalquraV1> for crate::SourceDataProvider {
     fn load(&self, req: DataRequest) -> Result<DataResponse<CalendarHijriUmmalquraV1>, DataError> {
         self.check_req::<CalendarHijriUmmalquraV1>(req)?;
-        let cache = HijriUmmAlQura::new_always_calculating().build_cache(1317..1567);
+
+        // From https://github.com/unicode-org/icu/blob/1bf6bf774dbc8c6c2051963a81100ea1114b497f/icu4c/source/i18n/islamcal.cpp#L87
```

Comment:
thought: in the medium term we should icuexportdata this.

Reply:
I would prefer if we would download it from an official web site and not rely on the provenance of the ICU data that happens to be in the repo.

Reply:
Does there exist a public data file intended for downloads like this on the official site? The Jira history of how the ICU repo got the data suggests not.

Reply:
I would rather not have the UAQ website in the critical path of datagen, the IANA website is already a pain.

I don't even think this should be data-driven, it was just easiest to implement this in datagen instead of `icu_calendar.